js/src/lib/components/InferenceWidget/shared/WidgetInputSamples/WidgetInputSamples.svelte

```diff
@@ -1,13 +1,14 @@
-<script lang="ts" generics="T extends WidgetExample">
+<script lang="ts">
 	import type { WidgetExample } from "../WidgetExample";
+	type T = $$Generic<WidgetExample>;
```

Comment:
like `TOutput`, maybe a more descriptive name like `TWidgetInput` or `TWidgetExample` would make it clear

Reply:
you prefer this syntax to the `generics="T extends WidgetExample">` syntax BTW @SBrandeis? 

(i think both are equivalent no?)